cfg/18.09/definitions.yaml

```diff
@@ -0,0 +1,1824 @@
+      For example: Add the line below to the /etc/audit/audit.rules file:
+      -w /etc/default/docker -k docker
+      Then restart the audit daemon. For example: 
+      service auditd restart
+    scored: true
+  - id: 1.2.9
+    description: "Ensure auditing is configured for Docker files and directories - /etc/sysconfig/docker (Scored)"
+    audit: "auditctl -l | grep /etc/sysconfig/docker"
+    tests:
+      test_items:
+      - flag: "/etc/sysconfig/docker"
+        set: true
+    remediation: |
+      You should add a rule for the /etc/sysconfig/docker file. 
+      For example: Add the line below to the /etc/audit/audit.rules file:
+      -w /etc/sysconfig/docker -k docker
+      Then restart the audit daemon. For example: 
+      service auditd restart
+    scored: true
+  - id: 1.2.10
+    description: "Ensure auditing is configured for Docker files and directories -
+    /etc/docker/daemon.json (Scored)"
+    audit: "auditctl -l | grep /etc/docker/daemon.json"
+    tests:
+      test_items:
+      - flag: "/etc/docker/daemon.json"
+        set: true
+    remediation: |
+      You should add a rule for the /etc/docker/daemon.json file. 
+      For example: 
+      Add the line below to the /etc/audit/audit.rules file:
+      -w /etc/docker/daemon.json -k docker
+      Then restart the audit daemon. 
+      For example: 
+      service auditd restart
+    scored: true
+
+  - id: 1.2.11
+    description: "Ensure auditing is configured for Docker files and directories -
+    /usr/bin/docker-containerd (Scored)"
+    audit: "auditctl -l | grep /usr/bin/containerd"
+    tests:
+      test_items:
+      - flag: "/usr/bin/containerd"
+        set: true
+    remediation: |
+      You should add a rule for the /usr/bin/containerd file. 
+      For example: 
+      Add the line below to the /etc/audit/audit.rules file:
+      -w /usr/bin/containerd -k docker
+      Then restart the audit daemon. 
+      For example: 
+      service auditd restart
+    scored: true
+
+  - id: 1.2.12
+    description: "Ensure auditing is configured for Docker files and directories -
+    /usr/sbin/runc (Scored)"
+    audit: "auditctl -l | grep /usr/sbin/runc"
+    tests:
+      test_items:
+      - flag: "/usr/sbin/runc"
+        set: true
+    remediation: |
+      You should add a rule for /usr/sbin/runc file. 
+      For example:
+      Add the line below to the /etc/audit/audit.rules file: 
+      -w /usr/sbin/runc -k docker
+      Then restart the audit daemon. 
+      For example: 
+      service auditd restart
+    scored: true
+
+- id: 2
+  description: "Docker daemon configuration"
+  checks:
+  - id: 2.1
+    description: "Ensure network traffic is restricted between containers on the
+    default bridge (Scored)"
+    audit: "docker network ls --quiet | xargs docker network inspect --format '{{ .Name }}: {{ .Options }}'"
+    tests:
+      test_items:
+      - flag: "com.docker.network.bridge.enable_icc:true"
```

Comment:
The spec says we should check for this being set to false, which is not necessarily the same as checking that it's not set to true (what if it's not present at all?)

Reply:
Yet again, it is a test from the old version as well.
The test is like that because that is an easy way to check it either not true or not present at all